### PR TITLE
🚸 Fix TFT menu positioning

### PR DESCRIPTION
**Pull request by thinkyhead**
Fix menu positioning issues described in #25475 using changes proposed by @helakejr

### CODE REVIEW

**Comment by thinkyhead** [comment]
CC: @uncellon — Does these changes fit in alright with your recent changes?

File: Marlin/src/lcd/tft/ui_480x320.cpp [hunk 1]
```diff
@@ -623,6 +623,7 @@ struct {
 #define BTN_HEIGHT 52
 #define X_MARGIN 20
 #define Y_MARGIN 15
+#define Y_OFFSET TERN(HAS_UI_480x272, 28, 34)
```
Comment (thinkyhead) [inline Marlin/src/lcd/tft/ui_480x320.cpp:626]: e.g., Maybe we should just be using `FONT_LINE_HEIGHT` which may be 34, 26, or 24 depending on the UI height and the selected `TFT_FONT`.

**Comment by uncellon** [comment]
@thinkyhead 
One moment, now I will check

**Comment by uncellon** [comment]
I tested portrait (480x320) and landscape (320x480) orientations:

- Main screen - OK
- Menu screen with navigation arrows - OK
- Homing screen - partially. Unfortunately, I haven't had time yet to fix this screen in vertical orientation 🥺

**Comment by thinkyhead** [comment]
> Homing screen - partially. Unfortunately, I haven't had time yet to fix this screen in vertical orientation

Maybe @helakejr is in the mood to add some more improvements…?

**Comment by uncellon** [comment]
I think I'll fix it this week. I will take PR (https://github.com/MarlinFirmware/Marlin/pull/25207) as an example (the design of the vertical homing menu).

**Pull request merged**